engine/polyglot-api/src/test/scala/org/enso/polyglot/ModuleManagementTest.scala

```diff
@@ -35,13 +36,27 @@ class ModuleManagementTest extends AnyFlatSpec with Matchers {
     def writeMain(contents: String): Unit = {
       Files.write(pkg.mainFile.toPath, contents.getBytes): Unit
     }
-  }
 
-  val subject = "Modules Polyglot API"
+    def teardown(): Unit = {
+      Files
+        .walk(tmpDir.toPath())
+        .sorted(Comparator.reverseOrder())
+        .map(_.toFile)
+        .forEach(_.delete())
```

Comment:
While this looks good, we already have Apache Commons IO in dependencies which provides `FileUtils.deleteDirectory` which serves the same purpose - so maybe it will be better to just use that (especially as some tests are already using it).

```suggestion
      FileUtils.deleteDirectory(tmpDir)
```

(requires also `import org.apache.commons.io.FileUtils`)

Reply:
Cool, I was worried I would be adding a new dependency but that is not the case. Will fix that.